### PR TITLE
Librispeech Models

### PR DESCRIPTION
**Pull request by skoocda**
Ready to merge; PR is composed of two additions- Librispeech YAML file and download script. Both have been tested and are fully functional.

I deleted the included Estonian/Voxforge models locally, and just noticed that the model deletions were not reverted. Still getting used to git.

However, I do think this PR is a better form to distribute and maintain. I would argue that the models themselves are not a good inclusion for the repository due to their size, which is considerable for the very particular use case that they serve. It would be optimal to just include download scripts for the estonian and voxforge models instead.

Finally... this is my first ever open source pull request! Excited to make revisions and get it pulled :smile:

### CODE REVIEW

File: librispeech_english_nnet2.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# You have to download Librispeech "online nnet2" models in order to use this sample
+# Run download-librispeech-nnet2.sh in '/opt/models' to download them.
+use-nnet2: True
+decoder:
+    # All the properties nested here correspond to the kaldinnet2onlinedecoder GStreamer plugin properties.
+    # Use gst-inspect-1.0 ./libgstkaldionline2.so kaldinnet2onlinedecoder to discover the available properties
+    use-threaded-decoder: True
+    model : /media/kaldi_models/english/librispeech_nnet_a_online/final.mdl
```
Comment (alumae) [inline librispeech_english_nnet2.yaml]: Why the absolute paths here?

**Comment by alumae** [comment]
Thanks for the PR. 
However, it seems that you are using absolute paths in your yaml file. It's better to use relative paths in our case (I added a comment).

Also, please don't delete other models with this PR. Even when this might be a good idea, it should be done using a separate PR.

**Comment by skoocda** [comment]
I fixed the yaml file, and returned the models. Is this now ready to merge?

**Comment by alumae** [comment]
The PR is still polluted with files not related to Librispeech, so I cherrypicked the download script and the conf file and just added them to the master branch. Thanks a lot for your efforts.

**Pull request closed without merging**